Dispatcharr incomplete install with AMD GPU passthrough

### ✅ Have you read and understood the above guidelines?

yes

### 🔎 Did you run the script with verbose mode enabled?

Yes, verbose mode was enabled and the output is included below

### 📜 What is the name of the script you are using?

Dispatcharr

### 📂 What was the exact command used to execute the script?

bash -c "$(curl -fsSL https://raw.githubusercontent.com/community-scripts/ProxmoxVE/main/ct/dispatcharr.sh)"

### ⚙️ What settings are you using?

- [ ] Default Settings
- [x] Advanced Settings

### 🖥️ Which Linux distribution are you using?

Debian 13

### 📈 Which Proxmox version are you on?

ve-manager/9.1.6/71482d1833ded40a (running kernel: 6.17.13-1-pve)

### 📝 Provide a clear and concise description of the issue.

Dispatcharr work as expected but with no Graphics pass and when I go to the LXC shell to troubleshoot type simple commands like ls, apt and update all I get is  'command not found' 

### 🔄 Steps to reproduce the issue.

Step 1 :run the lxc script. Step 2: go to the lxc console and there is no commands available "ls, apt, update" maybe more

### ❌ Paste the full error output (if available).

````Installing:                     
  firmware-amd-graphics

Suggested packages:
  initramfs-tools

Summary:
  Upgrading: 0, Installing: 1, Removing: 0, Not Upgrading: 0
  Download size: 15.1 MB
  Space needed: 94.1 MB / 12.1 GB available

Get:1 http://deb.debian.org/debian trixie/non-free-firmware amd64 firmware-amd-graphics all 20250410-2 [15.1 MB]
Fetched 15.1 MB in 0s (74.2 MB/s)          
Selecting previously unselected package firmware-amd-graphics.
(Reading database ... 21982 files and directories currently installed.)
Preparing to unpack .../firmware-amd-graphics_20250410-2_all.deb ...
Unpacking firmware-amd-graphics (20250410-2) ...
Setting up firmware-amd-graphics (20250410-2) ...
Hit:1 http://security.debian.org trixie-security InRelease
Hit:2 http://deb.debian.org/debian trixie InRelease                                  
Hit:3 http://deb.debian.org/debian trixie-updates InRelease                          
Get:4 https://repo.radeon.com/rocm/apt/7.2 noble InRelease [2,603 B]
Get:5 https://repo.radeon.com/rocm/apt/7.2 noble/main amd64 Packages [60.9 kB]
Fetched 63.5 kB in 0s (814 kB/s)    
All packages are up to date.    
Hit:1 http://deb.debian.org/debian trixie InRelease
Hit:2 http://security.debian.org trixie-security InRelease                           
Hit:3 http://deb.debian.org/debian trixie-updates InRelease                          
Hit:4 https://repo.radeon.com/rocm/apt/7.2 noble InRelease
Get:5 https://repo.radeon.com/amdgpu/latest/ubuntu noble InRelease [3,185 B]
Get:6 https://repo.radeon.com/amdgpu/latest/ubuntu noble/main amd64 Packages [1,328 B]
Err:6 https://repo.radeon.com/amdgpu/latest/ubuntu noble/main amd64 Packages
  File has unexpected size (1330 != 1328). Mirror sync in progress? [IP: 2600:1408:c400:e::17cd:6a0b 443]
  Hashes of expected file:
   - Filesize:1328 [weak]
   - SHA256:29b870578a963ca4e9cd4fb0557d38171a8c0cca88e1a58f9903b08ad62df0d2
   - SHA1:822f327a13108d12fd668c28a13900aff6effa5b [weak]
   - MD5Sum:71ecd7dc5b012493e67940cf4124fe59 [weak]
  Release file created at: Mon, 09 Feb 2026 19:19:58 +0000
Fetched 3,185 B in 0s (36.0 kB/s)
Error: Failed to fetch https://repo.radeon.com/amdgpu/latest/ubuntu/dists/noble/main/binary-amd64/Packages.gz  File has unexpected size (1330 != 1328). Mirror sync in progress? [IP: 2600:1408:c400:e::17cd:6a0b 443]
   Hashes of expected file:
    - Filesize:1328 [weak]
    - SHA256:29b870578a963ca4e9cd4fb0557d38171a8c0cca88e1a58f9903b08ad62df0d2
    - SHA1:822f327a13108d12fd668c28a13900aff6effa5b [weak]
    - MD5Sum:71ecd7dc5b012493e67940cf4124fe59 [weak]
   Release file created at: Mon, 09 Feb 2026 19:19:58 +0000
Error: Some index files failed to download. They have been ignored, or old ones used instead.
  ✖️   in line 4643: exit code 100 (APT: Package manager error (broken packages / dependency problems)): while executing command  apt update

  --- Last 20 lines of log ---
[2026-03-10 20:26:58] [INFO] Setting up Container OS
[2026-03-10 20:26:58] [OK] Set up Container OS
[2026-03-10 20:26:58] [OK] Network Connected: 10.1.1.111 2600:4040:2a20:7000:be24:11ff:fed5:7c9b 
[2026-03-10 20:26:59] [OK] IPv4 Internet Connected
[2026-03-10 20:26:59] [OK] IPv6 Internet Connected
[2026-03-10 20:26:59] [OK] Git DNS: github.com:(✔️ ) raw.githubusercontent.com:(✔️ ) api.github.com:(✔️ ) git.community-scripts.org:(✔️ )
[2026-03-10 20:26:59] [INFO] Updating Container OS
Hit:1 http://deb.debian.org/debian trixie InRelease
Hit:2 http://security.debian.org trixie-security InRelease
Hit:3 http://deb.debian.org/debian trixie-updates InRelease
Reading package lists...
[2026-03-10 20:27:13] [OK] Updated Container OS
[2026-03-10 20:27:14] [INFO] Setup Hardware Acceleration
[2026-03-10 20:27:16] [OK] Detected GPU: Advanced Micro Devices, Inc. [AMD/ATI] Rembrandt [Radeon 680M] (rev c8) (AMD)
[2026-03-10 20:27:16] [INFO] Configuring: Advanced Micro Devices, Inc. [AMD/ATI] Rembrandt [Radeon 680M] (rev c8)
[2026-03-10 20:27:16] [INFO] Installing AMD GPU drivers
[2026-03-10 20:27:26] [INFO] Installing ROCm 7.2 compute stack
[2026-03-10 20:27:27] [ERROR] in line 4643: exit code 100 (APT: Package manager error (broken packages / dependency problems)): while executing command  apt update
  -----------------------------------

  ✖️   Installation failed in container 110 (exit code: 100)
 📋 Installation log: /tmp/dispatcharr-110-00fd65e9.log

  Error: APT: Package manager error (broken packages / dependency problems)

What would you like to do?

  1) Remove container and exit
  2) Keep container for debugging
  3) Retry with verbose mode (full rebuild)
  4) Repair APT/DPKG state and re-run install (in-place)````

### 🖼️ Additional context (optional).

````[2026-03-10 20:27:29] [ERROR] Installation failed in container 110 (exit code: 100)

================================================================================
PHASE 2: APPLICATION INSTALLATION (Container)
================================================================================
[2026-03-10 20:26:58] [INFO] Setting up Container OS
[2026-03-10 20:26:58] [OK] Set up Container OS
[2026-03-10 20:26:58] [OK] Network Connected: 10.1.1.111 2600:4040:2a20:7000:be24:11ff:fed5:7c9b 
[2026-03-10 20:26:59] [OK] IPv4 Internet Connected
[2026-03-10 20:26:59] [OK] IPv6 Internet Connected
[2026-03-10 20:26:59] [OK] Git DNS: github.com:(✔️ ) raw.githubusercontent.com:(✔️ ) api.github.com:(✔️ ) git.community-scripts.org:(✔️ )
[2026-03-10 20:26:59] [INFO] Updating Container OS
Hit:1 http://deb.debian.org/debian trixie InRelease
Hit:2 http://security.debian.org trixie-security InRelease
Hit:3 http://deb.debian.org/debian trixie-updates InRelease
Reading package lists...
[2026-03-10 20:27:13] [OK] Updated Container OS
[2026-03-10 20:27:14] [INFO] Setup Hardware Acceleration
[2026-03-10 20:27:16] [OK] Detected GPU: Advanced Micro Devices, Inc. [AMD/ATI] Rembrandt [Radeon 680M] (rev c8) (AMD)
[2026-03-10 20:27:16] [INFO] Configuring: Advanced Micro Devices, Inc. [AMD/ATI] Rembrandt [Radeon 680M] (rev c8)
[2026-03-10 20:27:16] [INFO] Installing AMD GPU drivers
[2026-03-10 20:27:26] [INFO] Installing ROCm 7.2 compute stack
[2026-03-10 20:27:27] [ERROR] in line 4643: exit code 100 (APT: Package manager error (broken packages / dependency problems)): while executing command  apt update````

--- a/misc/tools.func
+++ b/misc/tools.func
@@ -4653,8 +4653,8 @@ EOF
   # Environment (PATH + LD_LIBRARY_PATH)
   if [[ -d /opt/rocm ]]; then
     cat <<'ENVEOF' >/etc/profile.d/rocm.sh
-export PATH="\$PATH:/opt/rocm/bin"
-export LD_LIBRARY_PATH="\${LD_LIBRARY_PATH:+\$LD_LIBRARY_PATH:}/opt/rocm/lib"
+export PATH="$PATH:/opt/rocm/bin"
+export LD_LIBRARY_PATH="${LD_LIBRARY_PATH:+$LD_LIBRARY_PATH:}/opt/rocm/lib"
 ENVEOF
     chmod +x /etc/profile.d/rocm.sh
     # Also make available for current session / systemd services

